wizard_builder/models.py

```diff
@@ -3,21 +3,30 @@
 from polymorphic.models import PolymorphicModel
 
 from django import forms
-from django.core.exceptions import ObjectDoesNotExist
+from django.contrib.sites.models import Site
+from django.core.exceptions import ImproperlyConfigured, ObjectDoesNotExist
 from django.db import models
+from django.db.utils import OperationalError
 from django.utils.safestring import mark_safe
 
+from .managers import PageBaseManager
+
 
 def get_page_position():
     try:
         return PageBase.objects.latest('position').position + 1
-    except ObjectDoesNotExist:
+    except (ObjectDoesNotExist, OperationalError):
```

Comment:
Ideally this problem would be solved not by ignoring the operational error, but instead by setting the default attribute inside the migration with

`apps.get_model...`

there's an example of that here https://github.com/SexualHealthInnovations/callisto-core/blob/f0b8ca40d1ce6b6e90130a2b7ac203f2f53660bc/callisto/delivery/migrations/0010_email_notification_data_migration.py#L10